contracts/main/stablecoin-core/StableSwapModule.sol

```diff
@@ -20,24 +20,21 @@ contract StableSwapModule is PausableUpgradeable, ReentrancyGuardUpgradeable, IS
     using SafeToken for address;
 
     IBookKeeper public bookKeeper;
-    IStablecoin public stablecoin;
+    address public stablecoin;
     address public token;
-    
-    uint256 internal to18ConversionFactor;
```

Comment:
we don't store conversion factor, but read decimals from token's contract instead